centinel/models.py

```diff
@@ -45,6 +45,8 @@ def __init__(self, **kwargs):
         # type
 
         allowed_keys = {"username": "string", "is_vpn": bool,
+                        "registered_date": datetime,
```

Comment:
Sorry, I forgot what this piece of code does. Forget that- I know that it needs to be a datetime class so we can do type checking.